tinyurl/tinyurl.py

```diff
@@ -15,7 +15,7 @@ def run(args_dict):
 
     # iterate through each link
     tinys = [requests.get(URL, params={'url': link}).text for link in
-             links['link']]
+             links['Link']]
```

Comment:
Fantastic!

Reply:
yes, good idea. I had to keep changing this on my own.